src/services/ApiService/Dex/Dex.ts

```diff
@@ -95,8 +95,8 @@ export default class DexService {
     const { data }: { data: DexTrade } = await this.service.get('swap/v1/quote', {
       baseURL: this.getBaseURL(network.id),
       params: {
-        sellToken: sellToken.ticker,
-        buyToken: buyToken.ticker,
+        sellToken: sellToken.contractAddress ?? sellToken.ticker,
+        buyToken: buyToken.contractAddress ?? buyToken.ticker,
```

Comment:
why contract address or ticker?

Reply:
The API supports either addresses or tickers. Since all our ERC20 assets have an attached `contractAddress`, we can use that and for the very few assets that don't, we can use the ticker. The assets without `contractAddress` will be the base asset for the network.